### PR TITLE
System requirement page changes.

### PR DESCRIPTION
**Pull request by bhagyashreelhr**

### CODE REVIEW

File: pages/01.HDM Documentation/04.system-requirements/default.md [hunk 1]
```diff
@@ -55,30 +43,38 @@ Related document: For hardware and system requirements, refer to the [VMware Har
...
+    *   Port 7000-7010 - inbound and outbound
+    *   Port 8000-8010 - inbound and outbound
+    *   Post 32820 - inbound and outbound
+ 
+##### Network connectivity between on-premises and the cloud
+*   Network Link >= 1 Gbps
+*   Network Round Trip Time Latencies: &lt; 30 ms
+ 
+##### Hardware Requirements
+*   For HDM component requirements, check the deployment planning section as per your cloud end-point.
```
Comment (sumitkapoor) [inline pages/01.HDM Documentation/04.system-requirements/default.md:75]: Should there be link to point to the section ?

File: pages/01.HDM Documentation/04.system-requirements/default.md [hunk 2]
```diff
@@ -55,30 +43,38 @@ Related document: For hardware and system requirements, refer to the [VMware Har
...
 
+##### Network Environment
+*   Cloud HDM components should have access to the WAN network.
+*   Cloud VCenter and ESXi should be accessible on-premises at port 443.
+*   E1000E and VMXNet3 network adapter should be available in  the cloud vCenter.
+*   Network on which HDM components are deployed should have access to the cloud vCenter at port 443 and the cloud ESXi at port 443 and 902.
+*   Firewall rules for HDM cloud access:
+    *   Port 22 - inbound and outbound
+    *   Port 2379 - outbound 
+    *   Port 6000-6010 - inbound and outbound
```
Comment (sumitkapoor) [inline pages/01.HDM Documentation/04.system-requirements/default.md:65]: I think we should also state the use of these ports.

**Review by sumitkapoor — CHANGES_REQUESTED** [review]

File: pages/01.HDM Documentation/04.system-requirements/default.md (hunk 1 above)
Comment (bhagyashreelhr) [reply pages/01.HDM Documentation/04.system-requirements/default.md:75]: Okay. we can put. Their will be 3 links as per cloud end-point.

File: pages/01.HDM Documentation/04.system-requirements/default.md (hunk 2 above)
Comment (bhagyashreelhr) [reply pages/01.HDM Documentation/04.system-requirements/default.md:65]: Okay. will check.

**Comment by bhagyashreelhr** [comment]
Closing this one. Getting conflicts with other changes. Will open another with specific branch.

**Pull request closed without merging**